filter-effects/Overview.bs

```diff
@@ -336,7 +339,7 @@ The first filter function or <a element>filter</a> reference in the list takes t
 
 A computed value of other than ''filter/none'' results in the creation of a <a href="https://www.w3.org/TR/CSS21/zindex.html">stacking context</a> [[!CSS21]] the same way that CSS 'opacity' does. All the elements descendants are rendered together as a group with the filter effect applied to the group as a whole.
 
-The 'filter' property has no effect on the geometry of the target element's CSS boxes, even though 'filter' can cause painting outside of an element's border box.
+The 'filter' property has no effect on the geometry of the target element's CSS boxes, although 'filter' can contribute to the element's [=ink overflow area=].
```

Comment:
CSS specs style: put a newline after each sentence or sentence fragment, not just at 80 columns. In this case, the newline would be after "CSS boxes", because there is a comma. The rationale is that it tends to make future edits easier to read because their highlighting in the code review diff tool shows just an edit to a few fragments as opposed to a change to the whole paragraph.

Reply:
Done.